presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -1458,7 +1458,6 @@ static OrcBatchRecordReader createCustomOrcRecordReader(
                 orcEncoding,
                 orcFileTailSource,
                 stripeMetadataSource,
-                Optional.empty(),
```

Comment:
Due to some unknown reason this was using the package protected method. I decided to move it to a public method instead of wrapping the stripeMetadataSource into the StripeMetadataSourceFactory.of(stripeMetadataSource)